topics/admin/tutorials/tool-management/tutorial.md

```diff
@@ -245,3 +259,20 @@ Interestingly, UseGalaxy.eu and UseGalaxy.org.au have different approaches:
 - Together, `usegalaxy.*` are working on a collaborative approach at [galaxyproject/usegalaxy-tools](https://github.com/galaxyproject/usegalaxy-tools) but this is not consumption ready yet.
 
 If running ephemeris directly is not your preference, there is also an Ansible [role](https://github.com/galaxyproject/ansible-galaxy-tools) and a sample [playbook](https://github.com/afgane/galaxy-tools-playbook) that can help automate some tasks.
+
+> ### {% icon tip %} What if environments are not working
+> It sometimes happens in Galaxy, that one environment isn't working anymore. It mostly happens from the start when it does happen. You can remove the environment on disk, or use the "Manage Dependencies" interface, select the environment, and delete it. Then re-install the dependency through the same Manage Dependencies interface.
+{: .tip}
+
+> ### {% icon tip %} Can you install multiple tools simultaneously?
+> Previous experience with this is not good, there was a lot of unsafe code that would do things simultaneously that would destroy config files. But now it's conda, so you may destroy conda environments. A solution for this is like how [usegalaxy.eu]()https://github.com/usegalaxy-eu/usegalaxy-eu-tools does it, where we keep the full list of tools we want to install, and then a CI server (jenkins) installs these. Here we can enforce that only a single install process is running at any time.
```

Comment:
```suggestion
> Previous experience with this is not good, there was a lot of unsafe code that would do things simultaneously that would destroy config files. But now it's conda, so you may destroy conda environments. A solution for this is like how [UseGalaxy.eu does it](https://github.com/usegalaxy-eu/usegalaxy-eu-tools), where we keep the full list of tools we want to install, and then a CI server (Jenkins) installs them. In this way we can enforce that only a single install process is running at any time.
```

Reply:
Feel free to just commit these changes directly :)